components/sidebar_right/sidebar_right.tsx

```diff
@@ -211,11 +223,7 @@ export default class SidebarRight extends React.PureComponent {
             content = (
                 <div className='post-right__container'>
                     <FileUploadOverlay overlayType='right'/>
-                    <RhsThread
-                        previousRhsState={previousRhsState}
-                        toggleSize={this.toggleSize}
-                        shrink={this.onShrink}
```

Comment:
I see these props are not requested / used in `RhsThread` props, so I removed them